### PR TITLE
Feature 1.2.5.2 Added service category class

### PR DESCRIPTION
**Pull request by jnakaso**
* Added method in clientService to get service-categories from DB
* Dropdown added to client UI to display/select service categories

Need improved component testing

### CODE REVIEW

File: pages/client/index.tsx [hunk 1]
```diff
@@ -8,29 +13,37 @@ import { Rating } from 'primereact/rating';
...
+import { ClientTicket, ServiceCategory, clientService } from '../../src/services/ClientService';
 import { Dropdown } from 'primereact/dropdown';
 
 const Client = () => {
     const { push } = useRouter();
+    // TODO create LoadingContext & Loading Indicator in layout
+    const [_loading, setLoading] = useState(false);
+    // TODO create customHook for serviceCategories
+    const [categories, setCategories] = useState<ServiceCategory[]>();
+    const [statuses, setStatuses] = useState<ServiceCategory[]>();
```
Comment (samhhenderson) [inline pages/client/index.tsx:25]: Is this type supposed to be ServiceCategory, or is it supposed to be ServiceStatus?

**Review by samhhenderson — COMMENTED** [review]
Looks like it all works. Minor question on type of statuses in client/index.ts. 

Bigger question not addressed elsewhere: Do we want to revisit naming and how our services are split up? Currently ClientService is doing pretty much everything. Would it be a good or bad idea to create a TicketService (or ServiceRequestService??) as a separate item from a ClientService. We'll be needing to create Clients (and Animals) and it might be a good idea to explicitly separate Clients and Tickets that way. Thoughts?

File: pages/client/index.tsx (hunk 1 above)
Comment (jnakaso) [reply pages/client/index.tsx]: Thanks, it's fixed.

**Review by samhhenderson — APPROVED** [review]
Looks good! (With the note as that we may restructure ClientService as discussed).

**Pull request merged**